src/DocumentFormat.OpenXml.Framework/DocumentFormat.OpenXml.Framework.csproj

```diff
@@ -14,21 +14,13 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Summary>The Open XML SDK provides tools for working with Office Word, Excel, and PowerPoint documents</Summary>
-    <Description>
-      <![CDATA[The Open XML SDK provides tools for working with Office Word, Excel, and PowerPoint documents. It supports scenarios such as:
-
-- High-performance generation of word-processing documents, spreadsheets, and presentations.
-- Populating content in Word files from an XML data source.
-- Splitting up (shredding) a Word or PowerPoint file into multiple files, and combining multiple Word/PowerPoint files into a single file.
-- Extraction of data from Excel documents.
-- Searching and replacing content in Word/PowerPoint using regular expressions.
-- Updating cached data and embedded spreadsheets for charts in Word/PowerPoint.
-- Document modification, such as removing tracked revisions or removing unacceptable content from documents.]]>
-    </Description>
-    <RootNamespace>DocumentFormat.OpenXml</RootNamespace>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
```

Comment:
Cool! didn't know this was a thing.

Can you put it in the Directory.Build.props for the src directory? Then all projects will have the README

Reply:
Good point, done 👍